release-team/role-handbooks/ci-signal/README.md

```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
   - To get a better and quicker feedback on upgrade/downgrade tests, we can look at upgrade jobs that have "parallel" in their title eg: [gce-1.10-master-upgrade-cluster-parallel](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-cluster-parallel). These complete faster and are more stable since they don't run slow long running tests. If the parallel upgrade/downgrade jobs are green, this indicates we are mostly good with respect to upgrade functionality
   - Yet another tip to assess seriousness of the failure is to see if both gce and gke counterparts of a upgrade/downgrade job are failing eg: [gce-1.10-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-master) and [gke-gci-1.10-gci-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gke-gci-1.10-gci-master-upgrade-master). If both are failing then its indicative of a systemic failure in upgrade functionality. If only gke jobs are failing then it might be a GKE only issue and does not block k8s release. In this case escalate to GKE Cluster Lifecycle team  (@roberthbailey, @krousey)
 
+## Working with SIGs outside sig-release
+2 scenarios that you will be involved in:
+
+1. Identifying tests from sig-<name> that should be/are part of sig-release's blocking and informing dashboards. Those tests could be submitted as part of a new feature/enhancement that sig-<name> is developing, or could be existing tests in blocking/informing dashboards.
+Questions to ask sig-<name>:
+  - Which e2e test jobs are release blocking for your SIG?
+  - What is the process for making sure the SIG's test grid remains healthy and resolving test failures?
+  - Would moving the e2e tests for the SIG into their own test jobs make maintaining those tests easier? If yes, consider placing them in a dashboard owned by sig-<name>.
+  - Is there a playbook for how to resolve test failures and how to identify whether or not another SIG owns the resolution of the issue? If not, could you (sig-<name>) develop one?
+  - What is the escalation point (email + slack) that will be responsible for keeping this test healthy?
+
+1. Escalating test failures/flakes to sig-<name>. Expectations:
+  - Each test must have an escalation point (email + slack).  The escalation point is responsible for keeping the test healthy.
```

Comment:
Where are these escalation points defined/enumerated

Reply:
Good point 🤔 I'm thinking:
- Once https://github.com/kubernetes/community/pull/3143 merges they will be in [kubernetes/community/devel/testing.md](https://github.com/kubernetes/community/blob/2720cc0ae1b1bd4a16dbf89c90a4d4393fa299a9/contributors/devel/testing.md) > `Escalating failures to a SIG`. I'll add a todo to link from the CI signal handbook to the contributor guide once that's done.
- Once work for https://github.com/kubernetes/sig-release/issues/441 is complete, `sig-release`'s tests will have more explicit owners+contact details assigned. I'll add another todo to link to `k/test-infra/testgrid/config.yaml` post https://github.com/kubernetes/sig-release/issues/441 closure.

Thoughts?

Reply:
It's a start, it still feels like it's spread around a few places.  How am I going to find an e-mail for a SIG if I'm trying to ping them about a specific test case?

Maybe it'd be better to say each test must be owned by a SIG, and the SIG is responsible for providing escalation points.  If a SIG doesn't provide specific escalation points for resolving test failures, the default would be general contact info for the SIG: its chairs, their slack channel, their mailing list.  Where should the SIG detail escalation points? Maybe sigs.yaml or its readme?

In terms of non-defaults, the only things we'll have soon are:
- For release-blocking jobs, the owning SIG and mailing list is determined by the alert options in testgrid's config.